src/main/java/org/opensearch/ltr/stats/suppliers/utils/StoreUtils.java

```diff
@@ -75,7 +76,7 @@ public String getLtrStoreHealthStatus(String storeName) {
                 clusterService.state().getRoutingTable().index(storeName)
         );
 
-        return indexHealth.getStatus().name().toLowerCase();
+        return indexHealth.getStatus().name().toLowerCase(Locale.ROOT);
```

Comment:
Not for this line.
Why do we use default locale here https://github.com/opensearch-project/opensearch-learning-to-rank-base/blob/main/src/main/java/com/o19s/es/termstat/TermStatQueryBuilder.java#L154-L155